lma-meetingassist-setup-stack/README_QBUSINESS.md

```diff
@@ -0,0 +1,28 @@
+# Amazon Q Business integration in LMA
+
+Amazon Q is a new generative AI-powered application that helps users get work done. Amazon Q can become your tailored business expert and let you discover content, brainstorm ideas, or create summaries using your company’s data safely and securely. For more information see: [Introducing Amazon Q, a new generative AI-powered assistant](https://aws.amazon.com/blogs/aws/introducing-amazon-q-a-new-generative-ai-powered-assistant-preview). This feature allows your users to converse with Amazon Q using Live Meeting Assistant to ask questions and get answers based on company data.
+
+## Deploy Amazon Q (your business expert) as a fallback source of answers, using Lambda hooks with CustomNoMatches/no_hits
+
+1. Before proceeding, you will need an existing deployment of a Q Business application. Please reference the AWS docs for creating a new [Q Business application](https://docs.aws.amazon.com/amazonq/latest/qbusiness-ug/create-application.html)
```

Comment:
It would help users that are new to Q Business provide some visuals or at a minimum what options to choose along the way. It wasn't clear to me whether an IDC user needed to be created during this step.

Reply:
I think I'm working under the assumption that you have a configured Q Business app already set up. I don't think we should hand hold on creating the Q Business app there is plenty of other docs around that setup.